app/models/ticket.model.js

```diff
@@ -0,0 +1,9 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var TicketSchema = new Schema({
```

Comment:
Unless we want tickets to be anonymous, the ticket should probably hold the email address of the user that created it.

Reply:
Also, the user schema should be modified to hold an array of tickets so that they can check up on them later to view the admin's response.

Reply:
i'll hang on to the array of tickets and keep it basic for now, since ticket number is the auto generated ID of the ticket field